test/functional/versioned-api.test.js

```diff
@@ -3,12 +3,13 @@
 const expect = require('chai').expect;
 const loadSpecTests = require('../spec/index').loadSpecTests;
 const runUnifiedTest = require('./unified-spec-runner/runner').runUnifiedTest;
-const ServerApiVersion = require('../../lib/core').ServerApiVersion;
+const ValidServerApiVersions = require('../../lib/core').ValidServerApiVersions;
```

Comment:
```suggestion
const validVersions = require('../../lib/core').ValidServerApiVersions;
```
Nit, but if you rename this here there's no need for the second declaration further down.